docs/commands.md

```
@@ -0,0 +1,454 @@
+        await interaction.reply(`Role: ${name}`);
+      },
+    },
+  },
+});
+```
+
+## File Organization
+
+For simple commands, a single file is fine:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)